handler/README.md

```diff
@@ -196,6 +209,16 @@ two independent tasks (graphs) with one node (action).
 See the [example section](#the-example) for a more complex scenario. Before we see the full 
 power of graphs, we need to understand how nodes are connected.
 
+####### Subtasks Processing Log
+
+Let's consider above subtasks definition. The processing log will look as follows.  
+
+| Task | Node alias | Status | Transition | Node Log |
+|---|---|---|---|---|
+| TASK | book-rest-api | SUCCESS | T1 | L-book-rest-api |
+| TASK | author-rest-api | SUCCESS | T2 | L-author-rest-api |
+| TASK | composite | SUCCESS | T3 |  |
```

Comment:
Can you please describe what is in log when we have nested composite nodes and if an error occurs.

Reply:
Subgraphs are evaluated, then at the end composite node id entry is added...